posts/2024-03-11-crates-io-download-changes.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "crates.io: Download changes"
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+[crates.io](https://crates.io) is the official package registry of the Rust community where developers can publish their Rust packages for others to use. It plays a crucial role in the Rust ecosystem by providing a centralized platform for package distribution.
+
+The Rust community has been growing rapidly, and with it, the number of downloads from crates.io has been increasing by 200% to 300% per year.
```

Comment:
are these numbers accurate or hallucinated?

Reply:
they are accurate. we are averaging above 200% growth and the growth from January to February was at almost 300%.